apps/comps/components/leaderboard-table/index.tsx

```diff
@@ -44,143 +46,157 @@ export function LeaderboardTable({
+                >
+                  Votes
+                </SortableTableHeader>
+                <TableHead className="flex justify-end pr-10 text-white">
+                  Profile
+                </TableHead>
+              </TableRow>
+            </TableHeader>
+            <TableBody>
+              {agents.map((agent) => {
+                const scoreSplit = agent.score.toString().split(".");
+                const scoreSize =
+                  scoreSplit.length > 0 ? scoreSplit[1]?.length || 0 : 0;
+
+                return (
+                  <TableRow
+                    key={agent.id}
+                    className="grid grid-cols-[1fr_1fr_2fr_1fr_1fr_1fr]"
                   >
-                    <AwardIcon size={17} />
-                    <span>3rd</span>
-                  </div>
-                ) : (
-                  <div className="mx-6 flex items-center justify-center rounded bg-gray-800 p-2">
-                    {agent.rank}
-                  </div>
-                )}
-              </TableCell>
+                    <TableCell className="xs:flex-row mr-10 flex flex-col items-center gap-2 py-6 pl-20">
+                      {agent.rank === 1 ? (
+                        <div
+                          className={cn(
+                            "flex w-20 items-center justify-center gap-1 rounded bg-[#594100] p-2 text-yellow-500",
+                          )}
+                        >
+                          <Trophy size={17} />
+                          <span>1st</span>
+                        </div>
+                      ) : agent.rank === 2 ? (
+                        <div
+                          className={cn(
+                            "flex w-20 items-center justify-center gap-1 rounded bg-gray-700 p-2 text-gray-300",
+                          )}
+                        >
+                          <AwardIcon size={17} />
+                          <span>2nd</span>
+                        </div>
+                      ) : agent.rank === 3 ? (
+                        <div
+                          className={cn(
+                            "flex w-20 items-center justify-center gap-1 rounded bg-[#1A0E05] p-2 text-[#C76E29]",
+                          )}
+                        >
+                          <AwardIcon size={17} />
+                          <span>3rd</span>
+                        </div>
+                      ) : (
+                        <div className="mx-6 flex items-center justify-center rounded bg-gray-800 p-2">
+                          {agent.rank}
+                        </div>
+                      )}
+                    </TableCell>
 
-              <TableCell className="flex items-center justify-start pr-10 text-gray-500">
-                {agent.score}
-              </TableCell>
+                    <TableCell className="flex items-center justify-start pr-10 text-gray-500">
+                      <BigNumberDisplay
```

Comment:
for the AgentRank, we want to display it as the whole integer/rounded value. but, we _do not_ want to use `BigNumberDisplay`. 

the scores will be anywhere from `0` to some number in the low thousands like `1500`. we want to display the full number and truncate decimals so that it's `0` or `100` or `1543`
<img width="791" alt="Screenshot 2025-06-18 at 5 56 37 PM" src="https://github.com/user-attachments/assets/89a0f1f8-6dc3-4865-9df4-e5e86e6261db" />